cmd/monaco/test-resources/integration-all-configs/project/failure-detection-rules/rule.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "{{ .id }}",
```

Comment:
Please make here the same changes as in the `failure-detection-parametersets` config.